CHANGELOG.md

```diff
@@ -2,4 +2,14 @@
 ---
 
 
-* 2020/10/2: First record in our changelog history 🎊 by [corp-0](https://api.github.com/users/corp-0) in PR #[5242](https://api.github.com/repos/unitystation/unitystation/pulls/5242)
+* 2020/10/8: Alien alloy can be used to craft beds, lockers, floor tiles, table frames, and (useless) airlock frames if you somehow get it. by [PetMudstone](https://github.com/PetMudstone) in PR #[5263](https://github.com/unitystation/unitystation/pull/5263)
+* 2020/10/8: Table frames can now be moved around. Added table frames made out of wood and alien alloy. New table frames are craftable. by [PetMudstone](https://github.com/PetMudstone) in PR #[5263](https://github.com/unitystation/unitystation/pull/5263)
+* 2020/10/8: Adds a few more kinds of tables. Bronze, alien, and fancy (carpeted) tables. Not yet constructable, sadly. by [PetMudstone](https://github.com/PetMudstone) in PR #[5263](https://github.com/unitystation/unitystation/pull/5263)
+* 2020/10/7: Added holster and briefcase of cash to uplink. by [PetMudstone](https://github.com/PetMudstone) in PR #[5259](https://github.com/unitystation/unitystation/pull/5259)
+* 2020/10/7: Fixed certain grass tiles not using the right sprite. by [PetMudstone](https://github.com/PetMudstone) in PR #[5259](https://github.com/unitystation/unitystation/pull/5259)
+* 2020/10/7: Added medal boxes that appear in the quartermaster's, head of security's, head of personnels, and research director's lockers. Sadly, you can't pin the medals on your chest. by [PetMudstone](https://github.com/PetMudstone) in PR #[5259](https://github.com/unitystation/unitystation/pull/5259)
+* 2020/10/7: Added a few more belts. by [PetMudstone](https://github.com/PetMudstone) in PR #[5259](https://github.com/unitystation/unitystation/pull/5259)
+* 2020/10/7: Added a sabre for the captain. by [PetMudstone](https://github.com/PetMudstone) in PR #[5259](https://github.com/unitystation/unitystation/pull/5259)
+* 2020/10/5: Touches up advanced tools: sounds and sprites are updated with state changes. Adds sounds for Jaws of Life and Drill. by [NoooneyDude](https://github.com/NoooneyDude) in PR #[5257](https://github.com/unitystation/unitystation/pull/5257)
+* 2020/10/3: Computers and their frames now have directional sprites. by [PetMudstone](https://github.com/PetMudstone) in PR #[5253](https://github.com/unitystation/unitystation/pull/5253)
+* 2020/10/2: First record in our changelog history 🎊 by [corp-0](https://github.com/corp-0) in PR #[5242](https://github.com/unitystation/unitystation/pull/5242)
```

Comment:
Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)

Reply:
Ignore this one

Reply:
Can you stop Codacy looking at specific files?

Reply:
Yep, I will make it ignore all .md’s because who cares if the style of a doc file has one space or two after a *